content/go/concepts/math-functions/terms/sin/sin.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'The Sin() Function in Go'
+Description: 'Learn how to use the Sin() function in Go to calculate the sine of an angle.'
+Subjects:
+  - 'Go'
+  - 'Math'
+Tags:
```

Comment:
Edit the Tags list so all the entries come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).